app/controllers/volunteers_controller.rb

```diff
@@ -72,7 +72,12 @@ def handle_volunteer_update
   end
 
   def invite_volunteer_user
-    User.new_volunteer(@volunteer) && @volunteer.user.invite!
+    return false if @volunteer.external
+    new_user = User.new(
+      email: @volunteer.contact.primary_email, password: Devise.friendly_token,
+      role: 'volunteer', volunteer: @volunteer
```

Comment:
IMHO it's more readable to put each key on its own line 👮

Reply:
IMHO that would be cool, but RuboCop, the very cop will definiteley complain about method length if I did that.